pkg/webui/components/button/index.js

```diff
@@ -154,49 +154,49 @@ const commonPropTypes = {
   /** The message to be displayed within the button. */
   message: PropTypes.message,
   /**
-   * A flag specifying whether the `danger` styling should applied to the
-   * button.
-   */
+  * A flag specifying whether the `danger` styling should applied to the
+  * button.
+  */
```

Comment:
I don't know how prettier would feel about this.

Reply:
Indeed. This is not common JSDoc styling either.